packages/admin-test-utils/src/index.ts

```diff
@@ -0,0 +1 @@
+export * as Fixtures from './fixtures';
```

Comment:
Let's make it lowercase, I'd normally opt for capitalised but because it's an object you might end up doing `FIXTURES.CONTENT_TYPES.ADDRESSES` which is hard to skim read due to nature of capitalisation lol